etna/search/tests/test_forms.py

```diff
@@ -4,12 +4,89 @@
 
 
 class CatalogueSearchFormTest(SimpleTestCase):
-    def test_start_date_after_end_date_is_invalid(self):
+    def test_empty_opening_start_date_is_valid(self):
         form = CatalogueSearchForm(
             {
                 "group": "tna",
-                "opening_start_date": "2000-01-01",
-                "opening_end_date": "1900-01-01",
+                "opening_end_date_0": "01",
+                "opening_end_date_1": "12",
+                "opening_end_date_2": "2002",
```

Comment:
```suggestion
                "opening_start_date_0": "01",
                "opening_start_date_1": "12",
                "opening_start_date_2": "2002",
```

Reply:
The case was correct, but Not relevant, code changed to subTest.